int32 block_id
---
bool result
